### ISSUE TITLE
repo metadata

### ISSUE THREAD

**Issue by hippi777** [void-linux/void-infrastructure#77]
hi there! :)

i think it could be nice to add some meta to the repositories, like last sync, location, custom packages, missing packages, supported architectures, clibs, and what not.. :D (plz ignore if i just said anything redundant)

in case of interest, i think single files for each would be the most universal format

(nothing urgent or anything, just some braindump before i forget to come up with the idea :D )

**Comment by the-maldridge** [comment]
I agree this is a good goal.  I'd like to actually have some small software that tracks this and just generates the pages.  Right now if a mirror falls off the sync, we don't know about it, but having a task that could track a file's propagation across the mirrors would help a lot.

I don't have bandwidth to do this right now with some other infrastructure projects going on, but I would like to tackle it in the future.  If someone else knows Go and would like to tackle this first, feel free to ping me.

**Comment by hippi777** [comment]
tldr: probably i wont be that lucky volunteer, or not soon...

i just arrived back here (while i just jumped over the other void notifications (packages and docs)), like i have this much load to try to reduce it bite-by-bite... also, im not into go lang, but it seems to be basically pretty simple, so most likely i wont be the one, but in the meantime im happy to assist if i can... in the meantime i hope i will be ready with the funds of my void installer sooner than later, and thats my priority now, but u will see when ill b there :D
